doc/runtimes/php.md

```diff
@@ -88,7 +88,7 @@ The following process is applied to determine how to start an app:
 # Version support
 
 The PHP project defines this [release schedule][]. Oryx supports all actively supported
-releases (7.2, 7.3, 7.4, 8.0, 8.1), in addition to 5.6 & 7.0.
+releases (7.2, 7.3, 7.4, 8.0, 8.1, 8.2), in addition to 5.6 & 7.0.
```

Comment:
Should we update this sentence? We're not supporting 7.4 and below, as they're now EOL (even though we're still hosting the images). My suggestion would be to go with: `Oryx supports all [actively supported releases](https://www.php.net/supported-versions.php).`
The link to the PHP website allows us not to keep track of all these in our own doc.

Reply:
Updated.